src/app/about/page.js

```diff
@@ -1,14 +1,13 @@
-import WhoWeAre from "@/components/about/WhoWeAre";
```

Comment:
You only need to edit for `Header.jsx` file for this issue. Please unstage this file from your branch then push changes. Thank you.